manager-rs/dauth-service/src/rpc/handlers/handler.rs

```diff
@@ -0,0 +1,8 @@
+use std::sync::Arc;
+
+use crate::data::context::DauthContext;
+
+/// Handles all RPC calls to the dAuth service.
+pub struct DauthHandler {
```

Comment:
What's up with the handlers::handler redirection? Will this DauthHandler (which is pretty empty now) be needed in the long run? It seems nicer and more consistent to me to have everything be at "handers::{NAME}", and this module sticks out as not following the pattern.

Reply:
For sure, I'll add separate handlers

Reply:
Okay, added the handlers